azkaban-web-server/src/main/java/cloudflow/models/ExecutionBasicResponse.java

```diff
@@ -6,28 +6,35 @@
 
 
 public class ExecutionBasicResponse {
-  private int executionId;
+  private String executionId;
   private String submitUser;
   private long submitTime;
-  private String experimentId;
   private String concurrentOption;
   private String failureAction;
   private boolean isNotifyFailureFirst;
   private boolean isNotifyFailureLast;
 
+  private String experimentId;
+  private String description;
+  private String previousFlowExecutionId;
+
   public ExecutionBasicResponse(ExecutableFlow executableFlow) {
     requireNonNull(executableFlow, "executable flow is null");
-    this.executionId = executableFlow.getExecutionId();
+    this.executionId = Integer.toString(executableFlow.getExecutionId());
     this.submitUser = executableFlow.getSubmitUser();
     this.submitTime = executableFlow.getSubmitTime();
     this.concurrentOption = executableFlow.getExecutionOptions().getConcurrentOption();
     this.failureAction = executableFlow.getExecutionOptions().getFailureAction().toString();
     this.isNotifyFailureFirst = executableFlow.getExecutionOptions().getNotifyOnFirstFailure();
     this.isNotifyFailureLast = executableFlow.getExecutionOptions().getNotifyOnLastFailure();
-    this.experimentId = "none"; //todo
+
+    //todo (sshardool): add these missing fields once available.
+    this.experimentId = "defaultExperimentId";
+    this.description = "defaultDescription";
+    this.previousFlowExecutionId = "defaultPreviousExecutionId";
```

Comment:
Can you include in the missing fields job execution attempts, runtime properties and partial run related  fields?

Reply:
Will be included in a subsequent PR to keep the scope of this one limited.

Reply:
We'll need those additions for the POC. Can you add them as a comment now to make sure we don't forget to do it?

Reply:
I believe all the TODOs left in the code have to be revisited before the POC itself.
Is the `todo` added here not sufficient ?